pydis_site/apps/resources/tests/test_resources.py

```diff
@@ -0,0 +1,9 @@
+from django.test import TestCase
+
+
+class TestResources(TestCase):
+    """Test our resource filtering systems."""
+
+    def test_utils_to_retrieve_tags(self):
+        """Test that the utils that retrieve the tags work as intended."""
+        pass
```

Comment:
Skipping this test?

Reply:
We should just delete this file. There are no utils to retrieve tags anymore.